broadcast/broadcast-client/client_builder_mocks.go

```diff
@@ -0,0 +1,39 @@
+package broadcast_client
+
+import (
+	broadcast_api "github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast/internal/arc/mocks"
+)
+
+type MockType int
+
+const (
+	MockSuccess MockType = iota + 1
```

Comment:
Why `+1`?

Reply:
I don't know, probably could or even should start from 0. For some reason I thought that enums start with 1